src/router.js

```diff
@@ -0,0 +1,14 @@
+const { homePage, publicHandler, notFound } = require("./handler");
+const router = (request, response) => {
+  const pathname = request.url;
+
+  if (pathname == "/") {
+    return homePage(request, response);
+  } else if (pathname.includes(".")) {
+    return publicHandler(request, response, pathname);
```

Comment:
I am not sure about how you checks the public URL, I think a better way to do it would be something like 
`if (['jpg','html','png','ico','js','css'].includes(pathname.split('.')[1])`

Reply:
@ghassanmas  we fix this condition